### ISSUE TITLE
Admin inaccessible with real_url_path error after Altis 5 upgrade

### ISSUE THREAD

**Issue by kadamwhite** [humanmade/altis-cms#296]
After upgrading to Altis 5, the admin of a site is failing to load with the following error:

```
Uncaught Error: Argument 1 passed to Altis\CMS\real_url_path() must be of the type string, null given, called in /usr/src/app/wordpress/wp-includes/class-wp-hook.php on line 287 in /usr/src/app/vendor/altis/cms/inc/namespace.php on line 352
Altis\CMS\real_url_path()
wp-includes/class-wp-hook.php:287
apply_filters()
wp-includes/plugin.php:206
apply_filters()
wp-includes/class.wp-scripts.php:328
do_item()
wp-includes/class.wp-dependencies.php:116
do_items()
wp-includes/class.wp-scripts.php:643
do_head_items()
wp-includes/script-loader.php:1841
print_head_scripts()
wp-includes/class-wp-hook.php:287
apply_filters()
wp-includes/class-wp-hook.php:311
do_action()
wp-includes/plugin.php:478
do_action()
wp-admin/admin-header.php:130
require_once()
wp-admin/index.php:111
```

Any page within wp-admin whitescreens with that visible trace (in a dev environment). Frontend of the site appears to function properly.

**Comment by roborourke** [comment]
This would happen if somehow a script or style is registered with a null value for the URL. Might there be something hooking into `script_loader_src` or `style_loader_src` that would do that?

**Comment by roborourke** [comment]
Fix released in v4.0.7 and v5.0.5
